src/libaktualizr/package_manager/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ aktualizr_source_file_checks(${SOURCES} packagemanagerconfig.cc ${HEADERS})
 target_sources(config PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/packagemanagerconfig.cc)
 
 add_aktualizr_test(NAME packagemanagerfake SOURCES packagemanagerfake_test.cc LIBRARIES PUBLIC uptane_generator_lib)
+add_test(NAME test_install_aktualizr_and_update COMMAND ${PROJECT_SOURCE_DIR}/scripts/test_install_aktualizr_and_update.sh ${PROJECT_BINARY_DIR})
```

Comment:
I'd suggest moving this line to `tests/CMakeLists.txt`. That's the generic place for miscellaneous tests, and this seems like an unexpected location to me.